### ISSUE TITLE
Please support Tagesgeld

### ISSUE THREAD

**Issue by tbm** [siddhantgoel/beancount-dkb#135]
DKB also has a savings product called "Tagesgeld".

The first line is:

```
"Konto";"Tagesgeld DE2xxxxxxxxxxxxxxxxxxx"
```

Everything else is the same.

If I replace `Tagesgeld` with `Girokonto` in the CSV, the importer works.

**Comment by siddhantgoel** [comment]
Sure, just to double check: does the ECImporter work for both old and new online banking interfaces? The header you added in the description seems to be from the newer interface which they launched quite recently.

**Comment by tbm** [comment]
The old format worked out of the box with beancount-dkb.

The format is:

```
"Kontonummer:";"DE2xxxxxxxxxxxxxxxxx / Tagesgeld";

"Von:";"01.01.2023";
"Bis:";"04.04.2023";
"Kontostand vom 04.04.2023:";"xx,xx EUR";

"Buchungstag";"Wertstellung";"Buchungstext";"Auftraggeber / Begünstigter";"Verwendungszweck";"Kontonummer";"BLZ";"Betrag (EUR)";"Gläubiger-ID";"Ma
ndatsreferenz";"Kundenreferenz";
...
```

**Comment by siddhantgoel** [comment]
Thanks. I just published v0.17.0 which might resolve this. Could you please double check?

**Comment by tbm** [comment]
Yes, I can confirm, thank you!

0.16.0 gave me no output with the new CSV format for `Tagesgeld` whereas 0.17.0 works as expected.

Thanks!
